app/models/notification.rb

```diff
@@ -68,7 +68,7 @@ def set_notification_recipients
 
   def set_custom_attributes
     return unless notification_blueprint # let validation catch this.
-    self.actions = UINotifications::URLResolver.new(
+    self.actions ||= UINotifications::URLResolver.new(
```

Comment:
I wonder if in this case we should extend the actions json attributes with additional data instead of hacking around subject? I understand that subject must be an AR object, but TBH I dont like the fact that we can't provide an official interface for plain ruby classes.